airbyte-cdk/python/airbyte_cdk/sources/streams/http/requests_native_auth/abstract_oauth.py

```diff
@@ -75,6 +76,30 @@ def build_refresh_request_body(self) -> Mapping[str, Any]:
 
         return payload
 
+    @property
+    def refresh_token_error_status_codes(self) -> Tuple[int]:
+        return (400,)
```

Comment:
```suggestion
        return ()
```

Reply:
let's leave this empty in the base class and let the connectors define their own status codes